conu/backend/docker/image.py

```diff
@@ -238,12 +239,16 @@ def _run_container(self, run_command_instance, callback):
             container_id = fd.read()
         return container_id, response
 
-    def run_via_binary(self, run_command_instance=None, *args, **kwargs):
+    def run_via_binary(self, run_command_instance=None, volume=None, *args, **kwargs):
         """
         create a container using this image and run it in background;
         this method is useful to test real user scenarios when users invoke containers using
         binary
 
+        :param volume: tuple or list of tuples in the form: target | source,target | source,target,mode
```

Comment:
I vote for more verbose description written as python code, something like:
```
 * `("/path/to/directory", )`
 * `("/host/path", "/container/path")`
```
please also make sure sphinx renders this nicely

Reply:
Done.